src/main/scala-2.11/org/apache/james/gatling/jmap/JmapAuthentication.scala

```diff
@@ -0,0 +1,54 @@
+package org.apache.james.gatling.jmap
+
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+class JmapAuthentication extends Simulation {
+
+  val domain:String = "domain-jmapauthentication.tld"
+  val username:String = "username@" + domain
+  val password:String = "password"
+
+  val httpProtocol = http
+    .baseURL("http://127.0.0.1")
+    .acceptHeader("application/json, text/plain")
+
+  def obtainContinuationToken(username: String) = exec(
+    http("obtainContinuationToken")
+      .post("/authentication")
+      .headers(Map(
+        "Content-Type" -> "application/json; charset=UTF-8",
+        "Accept" -> "application/json"))
```

Comment:
Do you still need this while you specified .acceptHeader("application/json, text/plain")?